sdk/core/scheduler/main.cc

```diff
@@ -124,8 +124,10 @@ namespace sched
 
 		for (size_t i = 0; auto *threadSpace : threadSpaces)
 		{
+			Debug::log("Created thread for trusted stack {}",
+			           info[i].trustedStack);
 			Thread *th = new (threadSpace)
-			  Thread(info[i].trustedStack, info[i].threadid, info[i].priority);
+			  Thread(info[i].trustedStack, i + 1, info[i].priority);
```

Comment:
This seems a little strange to me, since it means that the small-integer thread IDs are now assigned by link order (and will eventually be assigned by the outcome of sorting stacks for capalign packing), right?  If they show up in logging output, their mysterious sourcing might be unsettling?  (I don't think they show up in APIs, tho', so at least there's that!)